docs/03-for-team-members/02-becoming-a-team-member.md

```diff
@@ -0,0 +1,48 @@
+# Team members
+
+Flathub, as a centralized repository of software apps, has a dedicated team of
+contributors necessary to keep it running. These contributors are in charge of:
+
+* Reviewing new applications
+* Reviewing permission changes for existing aplications
+* Requesting permission changes for existent apps that might not fit the
```

Comment:
```suggestion
* Requesting permission changes for existing apps that might not fit the
```

Reply:
I think it is better to put "Reviewing existing applications on Flathub". It's not going to be only permissions.